src/main/java/at/fh/joanneum/irfc/rest/EventLocationApi.java

```diff
@@ -0,0 +1,63 @@
+package at.fh.joanneum.irfc.rest;
+
+import at.fh.joanneum.irfc.model.eventlocation.EventLocationDTO;
+import at.fh.joanneum.irfc.service.EventLocationService;
+
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+/**
+ * @author moe@softwaregaertner.at
+ **/
+@RequestScoped
+@Path("event-locations")
+public class EventLocationApi {
+
+    @Inject
+    EventLocationService eventLocationService;
+
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public List<EventLocationDTO> getAll() {
+        return eventLocationService.getAll();
+    }
+
+    @GET
+    @Path("{id}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public EventLocationDTO getById(@PathParam("id") Long id) {
+        return eventLocationService.getById(id);
+    }
+
+    @POST
```

Comment:
same with CreateOrUpdate?

Reply:
after discussion with Alex we now go with a put and a post endpoint for all classes